Integrations/DomainTools_Iris/iris.py

```diff
@@ -0,0 +1,393 @@
+from CommonServerPython import *
+import requests
+
+# disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' GLOBALS '''
+
+BASE_URL = 'http://api.domaintools.com'
+USERNAME = demisto.params().get('username')
+API_KEY = demisto.params().get('apikey')
+RISK_THRESHOLD = int(demisto.params().get('risk_threshold'))
+PIVOT_COUNT_THRESHOLD = int(demisto.params().get('pivot_count_threshold'))
+YOUNG_DOMAIN_TIMEFRAME = int(demisto.params().get('young_domain_timeframe'))
+
+''' HELPER FUNCTIONS '''
+
+
+def http_request(method, path, other_params=None):
+    """
+    HTTP request helper function
+    """
+    params = {'app_partner': 'demisto', 'app_name': 'Iris Plugin', 'app_version': '1', 'api_username': USERNAME,
```

Comment:
Please lint properly

Reply:
My linter doesn't see a problem here and the circle ci also didn't point anything out. Do you have an example of how you want this to look?